data-raw/scenario_geography_x_ald_sector.R

```diff
@@ -0,0 +1,53 @@
+# styler: off
+scenario_geography_x_ald_sector <- tibble::tribble(
```

Comment:
I would imagine we may have to extend this to add the scenario source at some point. Just as a pretext to what is likely to come in the future, obviously not needed now. But it is very unlikely that this mapping will e.g. also work for NGFS scenarios